lib/styleToFilters.js

```diff
@@ -173,29 +173,4 @@ function getPropertyFromExpression(exp, properties) {
   }
 }
 
-module.exports = styleToFilters;
-
-// cli tool
-if (require.main === module) {
-  var fs = require('fs');
-  var path = require('path');
-  var JSONStream = require('JSONStream');
```

Comment:
Note: this tool was broken as JSONStream was never added to the dependencies in package.json. So, I did not consider it important to keep the streaming functionality, as its clear it was not used.